waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx

```diff
@@ -1,5 +1,202 @@
 import React from 'react';
+import styled from 'styled-components';
+import LogoSVG from '../../../static/ste/ste-logo.svg';
+import EmailSVG from '../../../static/ste/icons/email.svg';
+import TwitterSVG from '../../../static/ste/icons/logo-twitter.svg';
+import InstagramSVG from '../../../static/ste/icons/logo-instagram.svg';
+import LinkedInSVG from '../../../static/ste/icons/logo-linkedin.svg';
+import FacebookSVG from '../../../static/ste/icons/logo-facebook.svg';
+import STEFooterImg from '../../../static/ste/background/footer-bg.svg';
 
-const STEFooter: React.FC = () => <div>STE Footer</div>;
+const fill = '#fff'; // Fill for text
+
+const FooterBackground = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  height: 9rem;
+  bottom: 0;
+  background: #3c8c4a;
+  padding: 2.5rem 6rem 7.5rem;
+  transition: all 0.3s;
+  border: none !important;
+  @media screen and (max-width: 425px) {
+    padding: 1rem;
+    min-height: 70vh;
+    overflow: hidden;
+  }
+`;
+
+const LogoAndContactWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+`;
+
+const Logo = styled.img.attrs({
+  src: `${LogoSVG}`,
+  alt: 'logo',
+})`
+  width: 260px;
+  height: 150px;
+  margin-bottom: -15px; // required to make up for slight gap in the svg
+`;
+
+const ContactDiv = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: start;
+`;
+
+const ContactUs = styled.p`
+  font-size: 18px;
+  margin: 10px 0;
+  color: ${fill};
+  font-weight: bold;
+`;
+
+const ContactInfo = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+`;
+
+const EmailIcon = styled.img.attrs({
+  src: `${EmailSVG}`,
+  alt: 'email',
+})`
+  width: 28px;
+  height: 24px;
+`;
+
+const ContactEmail = styled.p`
+  font-size: 14px;
+  margin: 0 5px;
+  color: ${fill};
+`;
+
+const Divider = styled.hr`
+  width: 100%;
+  border-color: ${fill};
+  margin: 10px 0;
+  border-width: 1px;
+`;
+
+const PageTagsAndSocialsWrapper = styled.div`
+  height: 70px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  @media screen and (max-width: 990px) {
+    flex-direction: column;
+  }
+`;
+
+const PageTagsWrapper = styled.div`
+  //   width: 30vw;
```

Comment:
Can you delete this instead of commenting?

Reply:
Change made in upcoming commit.